pkg/ui/react-app/src/thanos/pages/blocks/BlockDetails.tsx

```diff
@@ -40,6 +42,9 @@ export const BlockDetails: FC<BlockDetailsProps> = ({ block, selectBlock }) => {
     }
   };
 
+  const { response: flagsRes } = useFetch<FlagMap>(`/api/v1/status/flags`);
+  const disableAdminOperations = flagsRes?.data?.['disable-admin-operations'] || false;
```

Comment:
Let's actually call this in BlocksContent and pass it to BlocksDetails as a prop, that way we won't be calling this every time we open a block detail page. This also won't change from block to block.